nixos/modules/services/web-apps/mattermost.nix

```diff
@@ -756,7 +767,7 @@ in
           "d= ${tempDir} 0750 ${cfg.user} ${cfg.group} - -"
 
           # Ensure that pluginDir is a directory, as it could be a symlink on prior versions.
-          "r- ${pluginDir} - - - - -"
```

Comment:
These `r-` caused the warning. `d=` will make sure they're a directory instead of a symlink to one.